lib/bundle/cask_dumper.rb

```diff
@@ -16,13 +16,19 @@ def casks
             .uniq
     end
 
-    def dump(casks_required_by_formulae)
+    def dump(casks_required_by_formulae, describe: false)
+      to_line = ->(cask) { "#{desc_comment(cask)}cask \"#{cask}\"" }
```

Comment:
Could this be a dedicated method that has `desc_comment` inlined?

Reply:
It get awkward because you can't pass arguments (`describe`, in this case) to `map`… What exactly did you have in mind?

Reply:
Where/how does describe change behaviour? I don't see it being used here.

Reply:
> Where/how does describe change behaviour? I don't see it being used here.

Yeah I realised this, and fixed it… just forgot to push that change.